### PR TITLE
ADEN-5481 | Remove code after PremiumAdLayout roll-out

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5481

* prefooters
* skyscrappers
* modal_interstitial
* btfBlocker cleanup
* old floating medrec

### CODE REVIEW

File: extensions/wikia/Lightbox/LightboxHooks.class.php [hunk 1]
```diff
@@ -10,15 +10,7 @@ class LightboxHooks {
 	 * @return bool
 	 */
 	public static function onMakeGlobalVariablesScript( &$vars ) {
```
Comment (WikiaTech) [inline extensions/wikia/Lightbox/LightboxHooks.class.php:12]: MAJOR Don't use an OnMakeGlobalVariablesScript hook handler to register JS variables. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AOnMakeGlobalVariablesScript)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/js/context/slotsContext.js [hunk 2]
```diff
@@ -27,7 +27,6 @@ define('ext.wikia.adEngine.context.slotsContext', [
 		var context = adContext.getContext(),
 			isHome = context.targeting.pageType === 'home',
 			isOasis = context.targeting.skin === 'oasis',
-			isPremiumAdLayoutEnabled = context.opts.premiumAdLayoutEnabled,
 			isIncontentEnabled =
 				!isHome &&
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/context/slotsContext.js:31]: MAJOR Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js [hunk 3]
```diff
@@ -57,8 +46,6 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 	function createProvider(logGroup, providerName, src, slotMap, extra) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js:46]: MAJOR Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AAmdExcludedFunctionParameterLimit)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js [hunk 4]
```diff
@@ -24,41 +24,6 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.aol', [
...
-					sizeId: '154'
-				},
-				LEFT_SKYSCRAPER_3: {
-					size: [300, 600],
-					placement: '4431475',
-					network: network,
-					alias: '4431475',
-					sizeId: '529'
 				}
 				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js:28]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js [hunk 5]
```diff
@@ -11,7 +11,6 @@ define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
 
 	var logGroup = 'ext.wikia.adEngine.provider.gpt.adSizeFilter',
 		context = adContext.getContext(),
-		maxAdSize = 704,
 		minSkinWidth = 1240;
 
 	function isLargeBreakpoints() {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js]: MAJOR Remove unused function 'isLargeBreakpoints'. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 105 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 11 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L131](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L131): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L132](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L132): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L206](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L206): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/8b4ee7d76b39a4632d4b70150d5821674819811f/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 104 issues
* ![MAJOR][MAJOR] 88 major
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 11 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L131](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L131): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L132](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L132): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L206](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L206): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/36e893fa0ef38c8b5379df19ebd37625a05e2981/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: skins/wikia/WikiaMonoBook.php [hunk 6]
```diff
@@ -239,29 +239,4 @@ public function bottomScripts() {
...
-		}
-	}
-
-	function noSkyscraper() {
-		if ( $this->isSkyscraper() ) {
-			echo ' style="margin-right: 0px;"';
-		}
-	}
-
 } // end of class
```
Comment (WikiaTech) [inline skins/wikia/WikiaMonoBook.php:242]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 108 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 7 minor
* ![INFO][INFO] 12 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L131](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L131): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L132](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L132): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L206](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L206): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/19365d8da690c5f61d504b8a5060b64d929b7eab/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/Lightbox/scripts/LightboxLoader.js [hunk 7]
```diff
@@ -42,11 +42,6 @@
 				Lightbox.current.thumbs = [];
 				Lightbox.current.thumbTypesAdded = [];
 				Lightbox.to = LightboxLoader.cache.to;
-				// Reset Ad Flags
-				Lightbox.ads.adMediaProgress = [];
-				Lightbox.ads.adMediaShown = 0;
-				Lightbox.ads.adMediaShownSinceLastAd = 0;
-				Lightbox.ads.adIsShowing = false;
 				// Re-show box ad
 				LightboxLoader.pageAds.css('visibility', 'visible');
```
Comment (dianafa) [inline extensions/wikia/Lightbox/scripts/LightboxLoader.js:46]: what about this?

Comment (dianafa) [reply extensions/wikia/Lightbox/scripts/LightboxLoader.js:46]: ok, checked, this refers to `TOP_RIGHT_BOXAD`, it's ok

File: extensions/wikia/Rail/RailController.class.php [hunk 8]
```diff
@@ -101,13 +92,10 @@ protected function getLazyRail() {
...
 		} );
 
-		// ad mix experiment uses a wrapper to group recirculation and ad placeholder
-		if ( !$isPremiumAdLayoutEnabled ) {
-			$railLazyContent .= Html::element( 'div', [
-				'id' => 'WikiaAdInContentPlaceHolder',
-				'class' => 'rail-sticky-module'
-			] );
-		}
+		$railLazyContent .= Html::element( 'div', [
```
Comment (dianafa) [inline extensions/wikia/Rail/RailController.class.php]: are we sure? from what I see here it should happen when PAL **is not** enabled

File: extensions/wikia/Rail/scripts/Rail.js [hunk 9]
```diff
@@ -85,21 +83,13 @@ $(function () {
 						'wikia.window',
 						require.optional('ext.wikia.adEngine.adContext'),
 						require.optional('ext.wikia.adEngine.slot.premiumFloatingMedrec'),
-						require.optional('ext.wikia.adEngine.slot.floatingMedrec')
```
Comment (dianafa) [inline extensions/wikia/Rail/scripts/Rail.js]: is `premiumFloatingMedrec` something we'd like to rename ?

File: extensions/wikia/AdEngine/js/slot/floatingMedrec.js [hunk 10]
```diff
@@ -1,85 +1,133 @@
...
+		}
 
+		function hideRecirculation() {
+			recirculationElement.style.display = 'none';
+		}
 
-				$adSlot.css({
-					visibility: 'hidden'
+		function refreshAd() {
+			return new Promise(function (resolve, reject) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/slot/floatingMedrec.js:85]: MAJOR Remove the unused function parameter "reject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 63 issues
* ![MAJOR][MAJOR] 44 major
* ![MINOR][MINOR] 7 minor
* ![INFO][INFO] 12 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L131](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L131): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L206](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L206): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L230](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L230): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdLogicPageDimensions.js#L124](https://github.com/Wikia/app/blob/67e45debe6d324c7865c70ff0bdd0f59ae14d017/extensions/wikia/AdEngine/js/AdLogicPageDimensions.js#L124): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANestedIfDepth)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by fraszczakszymon — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 63 issues
* ![MAJOR][MAJOR] 44 major
* ![MINOR][MINOR] 7 minor
* ![INFO][INFO] 12 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L133](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L133): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L208](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L208): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L232](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L232): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L232](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L232): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdLogicPageDimensions.js#L124](https://github.com/Wikia/app/blob/2f2f04a7665288a8a8fbf4abbeecb004c77bd0ec/extensions/wikia/AdEngine/js/AdLogicPageDimensions.js#L124): Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANestedIfDepth)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**